src/rasmus_fuel/tests/test_fuel_emission_containership.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pytest
+
+from rasmus_fuel.fuel_emission_containership import (
+    convert_emission_kgpermeter_kgperNM,
+    convert_fuel_consumption_tonnsperday,
+    emission_CO2_diesel_MANandBW,
+    fuel_consumption_diesel_MANandBW,
+)
+
+
+def test_fuel_consumption_diesel_MANandBW_no_engine():
+    """Check for no power if no movement is happening."""
+    assert 0 == fuel_consumption_diesel_MANandBW(
+        speed_ship_og=10.0,
+        engine_power=0,
+    )
+
+
+def test_fuel_consumption_diesel_MANandBW_errors():
+    """Check if correct errors are raised"""
+    speed_ship = np.array([1, 2, 3])
+    power = np.array([1, 2])
+    with pytest.raises(ValueError) as execinfo:
+        fuel_consumption_diesel_MANandBW(
+            speed_ship_og=speed_ship,
+            engine_power=power,
+        )
+    assert str(execinfo.value) == 'Shape of speed_ship_og and engine_power need to agree.'
+
+
+def fuel_consumption_diesel_MANandBW_kwargs_ignored():
+    """Check that arbitrary kwargs are ignored."""
+    np.testing.assert_equal(
+        fuel_consumption_diesel_MANandBW(
+            speed_ship_og=1,
+            engine_power=1,
+        ),
+        fuel_consumption_diesel_MANandBW(
+            speed_ship_og=1,
+            engine_power=1,
+            nonexisting_kwarg=1234,
+        ),
+    )
+
+
+def test_emission_CO2_diesel_MANandBW_zero_fuel_consumption():
+    """Check for zero emission in case of zero fuel consumption provided."""
+    assert 0 == emission_CO2_diesel_MANandBW(
+        fuel_consumption=0,
+        sailing_time=3600.0,
+    )
+
+
+def test_emission_CO2_diesel_MANandBW_ship_at_port():
+    """Check for zero emission if sailing time is zero (vessel at port condition)."""
+    assert 0 == emission_CO2_diesel_MANandBW(
+        fuel_consumption=23.0,
+        sailing_time=0,
+    )
```

Comment:
I'm not sure I understand this. Why is sailing time zero an indication for vessel in port? Isn't that just saying that if the duration of the reporting period is zero, there is no emission during that time?

Reply:
You are write, it includes reporting period zero , I will correct this

Reply:
Yes, you are right. I will correct this, the test is meant to check for the zero reporting period .